### PR TITLE
set crowding penalty as a parameter

### PR DESCRIPTION
**Pull request by danrlu**
### Summary:
- **What: This PR will**: 
  + Consolidate the places we download `ncov` to only the Docker image. B/c it is in the root in the Docker, change all code that refers to `ncov` location to `/ncov`
  + Pin the version in the Docker image to the current newest, and in the future we'll do a monthly update of the pin
  + Add crowding penalty as a parameter in our builds.yaml
  + Secretly tag along: add nano to tree docker

- **Why**: Nextstrain recently did 2 changes related to crowding penalty: 
  + The crowding penalty was surfaced as a parameter in the main workflow [here](https://github.com/nextstrain/ncov/commit/3544dfd7df44697e231582bd96d3045fabc295c0).
  + There was a [fix](https://github.com/nextstrain/ncov/pull/772) in the way it was applied. In below, blue samples are focal samples and we tried to pull green contextual samples based on their proximity to blue samples with varying ways to apply crowding penalty. In the following 3 settings, the green samples tend to land on the left-er side of the tree (aka more closely related to the blue samples).
     **Here is before the change (default crowding penalty 0.01):** 
image
    **Here is after the change (default 0.1):**
image
    **Here is after the change and set it to 0:**
image
    So with the change, with default parameters, we are pulling samples that are more closely related to the focal samples while still reducing fully identical/stacking samples. Would be great to have this change in our workflow. 

### Demos:
Changes in Docker image and `nextstrain-dbg.wdl` worked in my tree EC2, once merged I will do another test run on staging. 

### Notes:
Thanks Jess!!

### Checklist:
- [x] I merged latest `<base branch>`
- [x] I manually verified the change
- [x] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/backend/Dockerfile.nextstrain [hunk 1]
```diff
@@ -36,7 +37,7 @@ RUN mkdir /ncov && \
     cd /ncov && \
     git init && \
     git remote add origin https://github.com/nextstrain/ncov.git && \
-    git fetch origin master && \
+    git fetch origin 28b48c86f3b967b961663477e97227f416c53da4 && \
     git reset --hard FETCH_HEAD
```
Comment (danrlu) [inline src/backend/Dockerfile.nextstrain:41]: not sure whether need to change any other parts of git command

**Comment by danrlu** [comment]
@jgadling please help! After cloning `ncov` in the Docker image, we also need to replace [this line](https://github.com/nextstrain/ncov/blob/master/workflow/snakemake_rules/main_workflow.smk#L492) with `        crowding = config["crowding"]["crowding_penalty"],` (sorry indentation was eaten XD) 
I'll delete the current [patch](https://github.com/chanzuckerberg/aspen/tree/trunk/src/backend/aspen/workflows/nextstrain_run/patches) file if not recycled. 
Thanks!!

**Comment by jgadling** [comment]
> @jgadling please help! After cloning `ncov` in the Docker image, we also need to replace [this line](https://github.com/nextstrain/ncov/blob/master/workflow/snakemake_rules/main_workflow.smk#L492) with ` crowding = config["crowding"]["crowding_penalty"],` (sorry indentation was eaten XD) I'll delete the current [patch](https://github.com/chanzuckerberg/aspen/tree/trunk/src/backend/aspen/workflows/nextstrain_run/patches) file if not recycled. Thanks!!

I added a commit to update this -- try it out?

**Comment by danrlu** [comment]
Fix: in the Docker image, 'ncov' is cloned to the root '/ncov', so had to change the folder path in all scripts that needs `ncov`
image

**Review by jgadling — APPROVED** [review]

**Pull request merged**